templates/types/streaming/fastapi/app/api/routers/models.py

```diff
@@ -80,9 +83,20 @@ def get_last_message_content(self) -> str:
         """
         Get the content of the last message along with the data content if available
         """
-        message_content = self.messages[-1].content
-        if self.data:
-            message_content += "\n" + self.data.to_raw_content()
+        if len(self.messages) == 0:
```

Comment:
this mixes the concerns: of getting the last message, converting it from vercel/ai to llamaindex format and looking up the annotation to use.
my guess is that  `looking up the annotation` is something that users might want to modify and also the comment in the TS code why we do it seems helpful to me